backend/src/main/java/com/google/models/Alert.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.models;
+
+import com.google.models.Anomaly;
+import com.google.models.Timestamp;
+import java.util.List;
+
+
+/** Store alert-related data. */
+public final class Alert {
+  
+  private final Timestamp timestampDate;
+  private final List<Anomaly> anomalies;
```

Comment:
just wanted to reiterate what we talked about today - List of longs to store anomalyIds

Reply:
the thing is we will only know the `anomalyIds` when we store the entities into the datastore, so i think maybe we should still keep the `List<Anomaly> anomalies` field in an alert. 

i think when we're storing `Alert` into the datastore like in the `Alert->AlertEntity` conversion, we can generate a list of `anomalyIds` and store that as a field in the `Alert` entity in the datastore. and when we're doing `AlertEntity->Alert` conversion, we will use the `anomalyIds` associated with an `AlertEntity` to find the associated anomalies and regenerate the `List<Anomaly> anomalies` for an `Alert` object. 

does this make sense?

Reply:
yup that makes sense! when you start writing the code for storing as an entity, probably will be good to make a comment to clarify that the list of anomalies is simply stored as their entity ids in the Datastore